nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -136,32 +136,28 @@ in {
       default = config.services.nextcloud.home;
       defaultText = literalExpression "config.services.nextcloud.home";
       description = lib.mdDoc ''
-        Data storage path of nextcloud.  Will be [](#opt-services.nextcloud.home) by default.
-        This folder will be populated with a config.php and data folder which contains the state of the instance (excl the database).";
+        Nextcloud's data storage path.  Will be [](#opt-services.nextcloud.home) by default.
+        This folder will be populated with a config.php file and a data folder which contains the state of the instance (excluding the database).";
       '';
       example = "/mnt/nextcloud-file";
     };
     extraApps = mkOption {
       type = types.attrsOf types.package;
       default = { };
       description = lib.mdDoc ''
-        Extra apps to install. Should be an attrSet of appid to packages generated by fetchNextcloudApp.
+        Extra apps to install. Should be an attrSet of appid to packages generated by fetchzip.
```

Comment:
I'd prefer to actually use `fetchNextcloudApp` here: first of all, this is the build we also use for apps packaged in nixpkgs. Second it supports additional useful stuff such as appending patches.

Reply:
Oh okay, done